### ISSUE TITLE
VoiceOver does not read onTapHint semantics labels

### ISSUE THREAD

**Issue by perclasson** [flutter/flutter#49698]
## Steps to Reproduce

1. Create a semantics node with onTapHint:
```dart
Semantics(
    onTapHint: 'show movies',
    onTap: () {},
    button: true,
    child: Text('Add'),
)
```
2. Run the app in iOS, for VoiceOver the onTapHint is not read out loud (even when "Speak Hints" is enabled under Verbosity). For TalkOver on Android the label is read. If this discrepancy is intended I think it should be [documented for clarity](https://api.flutter.dev/flutter/semantics/SemanticsHintOverrides/onTapHint.html). This was tested on physical devices, see the doctor logs below.

<details>
  <summary>Logs</summary>

```
[✓] Flutter (Channel master, v1.14.4-pre.82, on Mac OS X 10.15.2 19C57, locale en-DE)
    • Flutter version 1.14.4-pre.82 at /Users/perc/development/flutter
    • Framework revision 1a3379d571 (2 days ago), 2020-01-27 16:13:02 -0800
    • Engine revision c4229bfbba
    • Dart version 2.8.0 (build 2.8.0-dev.5.0 fc3af737c7)

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/perc/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-stable, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C505
    • CocoaPods version 1.8.4

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio
    • Android Studio at /Applications/Android Studio with Blaze.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 42.1.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[!] VS Code (version 1.41.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    ✗ Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (5 available)
    • Pixel 4    • 9B011FFAZ00072            • android-arm64  • Android 10 (API 29)
    • iPhone     • 00008030-0016518E34D2802E • ios            • iOS 13.2.3
    • macOS      • macOS                     • darwin-x64     • Mac OS X 10.15.2 19C57
    • Chrome     • chrome                    • web-javascript • Google Chrome 79.0.3945.130
    • Web Server • web-server                • web-javascript • Flutter Tools
```

</details>

**Comment by perclasson** [comment]
https://api.flutter.dev/flutter/semantics/SemanticsHintOverrides-class.html Never mind, I see here that this is intended behaviour:
https://api.flutter.dev/flutter/semantics/SemanticsHintOverrides-class.html
